src/Command/PluginCommand.php

```diff
@@ -369,6 +369,15 @@ public function buildOptionParser(ConsoleOptionParser $parser): ConsoleOptionPar
         ])->addOption('composer', [
             'default' => ROOT . DS . 'composer.phar',
             'help' => 'The path to the composer executable.',
+        ])->addOption('force', [
+            'short' => 'f',
+            'boolean' => true,
+            'help' => 'Force overwriting existing files without prompting.',
+        ])->addOption('theme', [
+            'short' => 't',
+            'help' => 'The theme to use when baking code.',
+            'default' => Configure::read('Bake.theme') ?? '',
+            'choices' => $this->_getBakeThemes(),
```

Comment:
This looks good. Seems like there is one small style issue that needs to be fixed and this is would be good to go.

Reply:
Updated for my style issue. Still another style issue in the plugin, but not related to this change.

Reply:
Style issues all resolved / suppressed.

I have another issue that relates to baking plugins, and requires a new option. Should I add that code to this PR, or create a new one? Sorry for bouncing around, I was planning on making that a separate PR because its different functionality, and a different reason, but it's basically all the same code.

New option 'modify' (default to true), to modify the Application.php file. Croogo CMS uses its own Plugin loader, so we don't want the Application file modified automatically. This option would allow us to turn off that functionality.

Reply:
You can open a separate PR for that.